### ISSUE TITLE
Allure report generated via Jenkins failed to Load the Execution Video in Safari Browser

### ISSUE THREAD

**Issue by ViniSiva** [presidenten/wdio-video-reporter#18]
**Describe the bug**
Allure report generated via Jenkins failed to Load the Video [Execution Video].
Same code base running from local machine, loads the video in allure report.Same is not happening while running the test in Jenkins.
Execution Video attachment created in allure without loading the video.

**Log**
11:12:27  "Video" Reporter:
11:12:27 Using reporter config:[
11:12:27   "dot",
11:12:27   "spec",
11:12:27   "allure",
11:12:27   [
11:12:27     null,
11:12:27     {
11:12:27       "saveAllVideos": false,
11:12:27       "videoSlowdownMultiplier": 10,
11:12:27       "videoRenderTimeout": 5
11:12:27     }
11:12:27   ]
11:12:27 ]
11:12:27 
11:12:27 Using config:{
11:12:27   "debugMode": false,
11:12:27   "logLevel": "info",
11:12:27   "videoRenderTimeout": 5,
11:12:27   "outputDir": "_results_",
11:12:27   "allureOutputDir": "allure-results",
11:12:27   "rawPath": "rawSeleniumVideoGrabs",
11:12:27   "usingAllure": true,
11:12:27   "saveAllVideos": false,
11:12:27   "videoSlowdownMultiplier": 10,
11:12:27   "excludedActions": [],
11:12:27   "jsonWireActions": [
11:12:27     "url",
11:12:27     "forward",
11:12:27     "back",
11:12:27     "refresh",
11:12:27     "execute",
11:12:27     "size",
11:12:27     "position",
11:12:27     "maximize",
11:12:27     "click",
11:12:27     "submit",
11:12:27     "value",
11:12:27     "keys",
11:12:27     "clear",
11:12:27     "selected",
11:12:27     "enabled",
11:12:27     "displayed",
11:12:27     "orientation",
11:12:27     "alert_text",
11:12:27     "accept_alert",
11:12:27     "dismiss_alert",
11:12:27     "moveto",
11:12:27     "buttondown",
11:12:27     "buttonup",
11:12:27     "doubleclick",
11:12:27     "down",
11:12:27     "up",
11:12:27     "move",
11:12:27     "scroll",
11:12:27     "doubleclick",
11:12:27     "longclick",
11:12:27     "flick",
11:12:27     "location"
11:12:27   ]
11:12:27 }
11:12:27 
11:12:27 
11:12:27 Max waiting time: 5s
11:12:27 
11:12:27 --- Video /home/jenkins/workspace/PIT_WDIO_V5/_results_/add-to-cartpdp--Windows10_Chrome_v68-should-be-able-to-add-a-normal-sku-to-cart-lv--CHROME--07-18-2019--00-40-21-196.mp4 ---
11:12:27 Waiting for video to exist: 0s
11:12:27 Waiting for video to be ready: 0s
11:12:27 Waiting for video to be ready: 1s
11:12:27 Waiting for video to be ready: 2s
11:12:27 Waiting for video to be ready: 3s
11:12:27 
11:12:27 Generated:[
11:12:27   "/home/jenkins/workspace/PIT_WDIO_V5/_results_/add-to-cartpdp--Windows10_Chrome_v68-should-be-able-to-add-a-normal-sku-to-cart-lv--CHROME--07-18-2019--00-40-21-196.mp4"
11:12:27 ]
11:12:27 
11:12:27 
11:12:27 
11:12:27 Done!
11:12:27 
11:12:27 
11:12:27  "dot" Reporter:
11:12:27 F
11:12:27 
11:12:27  "spec" Reporter:

**To Reproduce**
Try running test via Jenkins with allure report configurations

**Expected behavior**
clicking on 'Execution video' should loads the video  in allure report generated via Jenkins..

**Environment (please complete the following information):**
 - **wdio-video-reporter version:** [ 1.4.4,]
 - **WebdriverIO version:** [5.10.7]
 - **Mode:** [WDIO Testrunner- mocha]
 - **If WDIO Testrunner, running sync/async:** [ sync]
 - **Node.js version:** [10.15.4]
 - **NPM version:** [6.4.1]
 - **Browser name and version:** [All browsers in Saucelabs]
 - **Platform name and version:** [All browsers in Saucelabs]

**Additional context**
This problem we are seeing only when test run from Jenkins.
Local testing load the video as expected.
Might be related the storing the video Artifacts for allure report to refer
<img width="700" alt="Screen Shot 2019-07-18 at 11 42 16 AM" src="https://user-images.githubusercontent.com/39404921/61433512-233a9e00-a951-11e9-89a0-d48f44a9e6d5.png">
.

**Comment by presidenten** [comment]
I will try to set this up and test it.

Until then, does it work if you run it on your local machine?

**Comment by presidenten** [comment]
I have verified that safari does not want to play videos from the jenkins allure report. 

It does however seem to be an allure issue, since the solution lies in modifying the generated html to to play nice with safaris autoplay policy.

See stackoverflow issue:
https://stackoverflow.com/questions/20347352/html5-video-tag-not-working-in-safari-iphone-and-ipad

No problems to view videos in chrome or firefox: https://imgur.com/a/sAbgSbx

Closing issue, since Allure need to do the fix.
